modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -33,7 +34,11 @@ def submit
           # evss_data = evss_mapper_service(auto_claim).map_claim
           # evss_claim = evss_service.submit(auto_claim, evss_data)
 
-          render json: auto_claim
+          if auto_claim
+            render json: auto_claim
+
+            build_pdf_payload # prepare the PDf to send off
```

Comment:
I would move this up. ultimately this will probably be moved to within one of the jobs. we'll only be using this in the 2nd and 3rd job for this flow so it will make sense to live there eventually..... but for now (to test it) we can just move it up and with our logging, we'll be able to confirm that we're getting a token and retrieving a token from redis as well. Better to test now while no one's really using the service.

Reply:
will do, making this update now